### PR TITLE
Bring childcare to the top

### PR DESCRIPTION
**Pull request by bvdmitri**
## Briefly Describe your changes

This PR brings the childcare info to the top of the website. This was an explicit request from Long. We got about 5 children per day where the maximum is 15.

## Checklist for merge
- [x] I built the website locally and tested it using `Franklin.serve()`
- [x] All CI checks have passed and you have tested the online version (click on the list of checks and click `Build and Deploy / Preview` to see the preview)
- [ ] Somebody else reviewed my changes (use your best judgement if you need to merge quickly

## After merge
- [ ] Closed relevant issues

### CODE REVIEW

**Comment by pabloferz** [comment]
I feel that the box makes it look out of place. Maybe we could have it as a regular section (we have the box at the end any way).

**Comment by bvdmitri** [comment]
Fair point, what about something like this ?
image

**Comment by giordano** [comment]
Maybe I should open a separate ticket about this, but it's weird that hovering over the image looks like they're links pointing to a separate page, but clicking on them does nothing:
image
Note the mouse pointer which turns into the click-link shape

**Comment by bvdmitri** [comment]
Yeah, we can remove those in a separate PR, probably some unnecessary CSS styling

**Comment by pabloferz** [comment]
> Fair point, what about something like this ? image

Yes. That's much better!

**Comment by vchuravy** [comment]
I was about the mention on https://github.com/JuliaCon/www.juliacon.org/pull/589 that the whole section is now pushed below "previous conference" making it unlikely that folks will actually see it.

File: 2024/index.md [hunk 1]
```diff
@@ -14,8 +14,26 @@ Co-hosted with <a style="color: #4366d0; text-decoration: underline" href="https
 top_title_heading = "../assets/2021/img/world_768.png 768w, ../assets/2021/img/world_1400.png 1400w, ../assets/2021/img/world_2800.png 2800w"
 +++
 
+Welcome to **JuliaCon 2024**, the premier annual conference for the Julia programming language community! This exciting event brings together developers, researchers, and enthusiasts from around the globe to celebrate and explore the power and versatility of Julia. 
+\\
+~~~
+<div class="row masonry" data-masonry='{ "itemSelector": ".grid-item", "percentPosition": true }'>
+~~~
+
+\begin{widebox}{title="Diversity Initiative & Childcare", color="purple"}
```
Comment (vchuravy) [inline 2024/index.md:23]: Maybe also mention the mentorship program added in https://github.com/JuliaCon/www.juliacon.org/pull/589

Comment (lochen906) [reply 2024/index.md:23]: Can we merge and publish ASAP? Only one week to go and I still don’t see it on the front page. Many thanks

Comment (bvdmitri) [reply 2024/index.md]: @lochen906 merged now

**Pull request merged**